app/routes/api/helpers.js

```diff
@@ -0,0 +1,14 @@
+const Joi = require('joi')
+
+const searchPayloadValidations = () => ({
+  offset: Joi.number().default(0),
+  limit: Joi.number().greater(0).default(20),
+  search: Joi.object({
+    text: Joi.string().valid().optional().allow(''),
+    type: Joi.string().valid().optional().default('sbi')
```

Comment:
@alex742  @Afshin-Karamifar-Capgemini  why we don't have a default to search by sbi in the new world in the old journey  the default is 'sbi'?

Reply:
We don't need default('sbi'), that should be type: Joi.string().valid().optional().allow('')